document/document_test.go

```diff
@@ -92,13 +91,9 @@ func TestFieldBuffer(t *testing.T) {
 			{"add field", `{"a": {"b": [1, 2, 3]}}`, `c`, document.NewTextValue("foo"), `{"a": {"b": [1, 2, 3]}, "c": "foo"}`, false},
 			{"nested doc", `{"a": "foo"}`, `a`, document.NewDocumentValue(document.NewFieldBuffer().
 				Add("b", document.NewArrayValue(document.NewValueBuffer().
-				Append(document.NewIntegerValue(1)).
-				Append(document.NewIntegerValue(2)).
-				Append(document.NewIntegerValue(3))))), `{"a": {"b": [1, 2, 3]}}`, false},
-			{"nested doc", `{"a": {"b": [1, 2, 3]}}`, `a.b`, document.NewArrayValue(document.NewValueBuffer().
```

Comment:
Could you explain this change? Ideally, we want to avoid changes that are irrelevant to the current PR. If this is important, let's open another PR, even if the change is small it's fine

Reply:
Mistakenly done. Sorry.

Reply:
This is marked as resolved but wasn't applied

Reply:
Now is done. Sorry